### PR TITLE
Slim down view mode dropdown by using icons

### PR DESCRIPTION
**Pull request by philippotto**
image

### URL of deployed dev instance (used for testing):
- https://viewmodeicons.webknossos.xyz

### Steps to test:
- use view mode UI to switch the modes

### Issues:
- helps with too wide toolbar content

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)

### CODE REVIEW

File: frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx [hunk 1]
```diff
@@ -51,24 +72,26 @@ class ViewModesView extends PureComponent<Props, {}> {
...
-          width: 120,
-        }}
-        onChange={this.handleChange}
-      >
-        {[
-          constants.MODE_PLANE_TRACING,
-          constants.MODE_ARBITRARY,
-          constants.MODE_ARBITRARY_PLANE,
-        ].map((mode) => (
-          <Option key={mode} disabled={this.isDisabled(mode)} value={mode}>
```
Comment (hotzenklotz) [inline frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx:67]: What happened to the `disabled` prop? The new dropdown will always show all the modes, right?

**Review by hotzenklotz — CHANGES_REQUESTED** [review]
Changes LGTM. Please see my comment about the `disabled` prop.

File: frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx (hunk 1 above)
Comment (philippotto) [reply frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx:67]: Oh, very good point :+1:

Comment (philippotto) [reply frontend/javascripts/oxalis/view/action-bar/view_modes_view.tsx:67]: fixed now!

**Review by hotzenklotz — APPROVED** [review]

**Pull request merged**